vmdb/app/controllers/api_controller.rb

```diff
@@ -66,8 +66,10 @@ class UnsupportedMediaTypeError < StandardError; end
   # Converted to @attr_<type> hashes at init, much faster access.
   #
   ATTR_TYPES = {
-    :time => %w(expires_on),
-    :url  => %w(href)
+    :time      => %w(expires_on),
+    :url       => %w(href),
+    :encrypted => %w(password) | ::MiqProvisionWorkflow.all_encrypted_options_fields.map(&:to_s) |
```

Comment:
of note, these are hardcoded in `FixAuth` logic.
Since it is standalone, can't really do anything about that.